src/api/views.py

```diff
@@ -59,12 +59,12 @@ def in_domains(domain):
     given domain doesn't end with .gov, ".gov" is added when looking for
     a match.
     """
-    domain = domain.lower()
+    Domain = apps.get_model("registrar.Domain")
     if domain.endswith(".gov"):
-        return domain.lower() in _domains()
+        return Domain.available(domain)
```

Comment:
**(Very Dumb Question):** I'm trying to figure out where exactly it does the check for case sensitivity...would you be able to point me to where?

From the `available` function in getgov -- I see `string_could_be_domain` for a regex, but when I go from `Domain.available` -> `commands.CheckDomain` (found in the `epplib` folder) -> ` _get_check_payload`

I'm assuming it'd be done in `_get_check_payload`, but I'm trying to find the logic that fully convinces me of the full check 😅

Reply:
Yes of course! Rachid made a series of tests in [test_models_domain.py](https://github.com/cisagov/getgov/blob/5c0e19397449d87815c70ebf2b896ffe66852945/src/registrar/tests/test_models_domain.py#L398) that verify that Domain.available -> commands.CheckDomain handle case sensitivity in domains - if you think it'd be good to add that case sensitivity in our API check I can also add that in in another PR!

Reply:
It's all good! I saw that you handled it in your PR tests (which is great!), was just trying to wrap my head around the logic of how CheckDomain in EPPLib did it -- EPPLib truly is magical 😆 Thanks for confirming!!